test/README.md

```diff
@@ -100,3 +103,11 @@ For all unit tests, please use the [shallow renderer](https://github.com/airbnb/
 If the Component being unit tested requires a DOM, you can use the [mount api](https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md) from `enzyme`. For some operations, you may still need to use the React test utils, but try to use the `enzyme` API as much as possible.
 
 Stick to test assertions such as `assert.strictEqual` and `assert.ok`. This helps keep tests simple and readable.
+
+#### Visual regression tests
+
+We should try to import as many demos from the documentation as possible.
+However, we can't replace one with the other, they address different needs.
```

Comment:
"We should try to import as many demos from the documentation as possible; 
however, we can't always replace one with the other as they address different needs."

(I had to check 😄 https://en.wikipedia.org/wiki/Conjunctive_adverb#English_punctuation)